R/combinations-calculate_matrix_metric.R

```diff
@@ -96,13 +96,13 @@ calculate_Bliss <- function(sa1, series_id1, sa2, series_id2, metric) {
   data.table::setnames(sa2, "x", "metric2", skip_absent = TRUE)
 
   # TODO: ensure they're unique?
```

Comment:
This task in comment is realized thanks to `expand.grid` /  `data.table::CJ` or not?

Reply:
Yes

Reply:
✅